### ISSUE TITLE
Inform python version

### ISSUE THREAD

**Issue by nelsonmarcos** [globocom/glog-cli#6]
Since python 3 is very popular right now, it would be helpful to inform the supported python version.

**Comment by sinvalmendes** [comment]
You're right, would be nice if you could do it for us, I think that keep this information at the README.md file should do it, what you think?

**Comment by nelsonmarcos** [comment]
Done: https://github.com/globocom/glog-cli/pull/7
